packages/imperative/src/rest/src/session/ConnectionPropsForSessCfg.ts

```diff
@@ -399,14 +399,14 @@ export class ConnectionPropsForSessCfg {
 
                 reasonForPrompts += "Therefore, you will be asked for the connection properties " +
                     "that are required to complete your command.\n";
-                connOpts.parms.response.console.log(TextUtils.wordWrap(
+                connOpts?.parms?.response?.console?.log(TextUtils.wordWrap(
```

Comment:
I'm not sure all these question marks are necessary, but they shouldn't hurt anything either 😋 

```suggestion
                connOpts?.parms.response.console.log(TextUtils.wordWrap(
```

Reply:
Parms is optional on connOpts, so we will need to at least keep that one. We do the same check on Line 387, so we would need to remove that check as well.